config/routes.rb

```diff
@@ -37,13 +37,15 @@
   delete "/reviews/:id", to: "reviews#destroy"
 
   post "/cart/:item_id", to: "cart#add_item"
+  patch "/cart/:item_id", to: "cart#change_quantity"
```

Comment:
Is it necessary to break from RESTful convention here?

Reply:
I was figuring that the above already broke restful (add_item is not restful), and restful has a patch and post verb to the same path, I just changed the action. Open to suggestions!

Reply:
I agree that they didn't really set us up for successfully using RESTful routing for the cart controller which is odd. I think this could be refactored **if** we wanted to go down that path.

In my mind we'd have to do something like Cart::ItemsController and then use the standard CRUD actions from there...so `add_item` would be `new`, `remove_item` would be `destroy` and `change_quantity` would be `update`

Reply:
I say we leave it to a refactor. I'll put this in a card and you can merge this.

Reply:
Updated!